### ISSUE TITLE
Pre-trained models?

### ISSUE THREAD

**Issue by anguyen8** [cfchen-duke/ProtoPNet#9]
Dear authors,

Thank you so much for releasing the code of this fascinating work!
Is there any chance you could also release the pre-trained ProtoPNet models used in your paper?

This would be very useful for us and the community, I believe. Thanks!

Anh

**Comment by Dadatata-JZ** [comment]
I would like to second this suggestion. I followed the instruction and the result is not reproducible (indeed, the testing accuracy I got was way off). Is there any tips for training protoPnet? @cfchen-duke Thank you in advance.

**Comment by anaghajoshi111** [comment]
> I would like to second this suggestion. I followed the instruction and the result is not reproducible (indeed, the testing accuracy I got was way off). Is there any tips for training protoPnet? @cfchen-duke Thank you in advance.

@Dadatata-JZ Can you tell me what versions of PyTorch, NumPy, cv2, and  Augmentor you used to get the accuracy?

**Comment by KodeRikke** [comment]
@Dadatata-JZ  Did you figure out this issue? We are having trouble reproducing the accuracy too.
